src/cpu.h

```diff
@@ -174,34 +177,121 @@ void OPC_ADD_A_d8(void);
+ * 	      the value stored in A and store it back into A.
+ */
 void OPC_SUB_A_A(void);
 
+/**
+ * @brief Subtract the value stored in Register B from
+ * 	      the value stored in A and store it back into A.
+ */
 void OPC_SUB_A_B(void);
 
+/**
+ * @brief Subtract the value stored in Register C from
+ * 	      the value stored in A and store it back into A.
+ */
 void OPC_SUB_A_C(void);
 
+/**
+ * @brief Subtract the value stored in Register D from
+ * 	      the value stored in A and store it back into A.
+ */
 void OPC_SUB_A_D(void);
 
+/**
+ * @brief Subtract the value stored in Register E from
+ * 	      the value stored in A and store it back into A.
+ */
 void OPC_SUB_A_E(void);
 
+/**
+ * @brief Subtract the value stored in Register H from
+ * 	      the value stored in A and store it back into A.
+ */
 void OPC_SUB_A_H(void);
 
+/**
+ * @brief Subtract the value stored in Register L from
+ * 	      the value stored in A and store it back into A.
+ */
 void OPC_SUB_A_L(void);
 
 /**
  * @brief First fetches a byte from the address `HL`,
- * 		  then subtracts the fetched byte from A.
+ * 	      then subtracts the fetched byte from A.
  */
 void OPC_SUB_A_HL(void);
 
 /**
  * @brief First fetches an immediate byte from PC + 1,
- * 		  then subtracts the fetched byte from A.
+ * 	      then subtracts the fetched byte from A.
  *
  * @warning PC cannot be incremented before this operation is completed
  * 			since it reads the data from the opcode itself.
  */
 void OPC_SUB_A_d8(void);
 
+
+
+/**
+ * @brief Subtract the value stored in Register A from the value stored in A, 
+ * 	      additionally subtract the carry flag, then store it in A.
+ */
+void OPC_SBC_A_A(void);
+
+/**
+ * @brief Subtract the value stored in Register B from the value stored in A,
+ *        additionally subtract the carry flag, then store it in A.
+ */
+void OPC_SBC_A_B(void);
+
+/**
+ * @brief Subtract the value stored in Register C from the value stored in A,
+ *        addtionally, subtract the carry flag, then store it in A.
+ */
+void OPC_SBC_A_C(void);
+
+/**
+ * @brief Subtract the value stored in Register D from the value stored in A,
+ *        addtionally, subtract the carry flag, then store it in A.
+ */
+void OPC_SBC_A_D(void);
+
+/**
+ * @brief Subtract the value stored in Register E from the value stored in A,
+ *        addtionally, subtract the carry flag, then store it in A.
+ */
+void OPC_SBC_A_E(void);
+
+/**
+ * @brief Subtract the value stored in Register H from the value stored in A,
+ *        addtionally, subtract the carry flag, then store it in A.
+ */
+void OPC_SBC_A_H(void);
+
+/**
+ * @brief Subtract the value stored in Register L from the value stored in A,
+ *        addtionally, subtract the carry flag, then store it in A.
+ */
+void OPC_SBC_A_L(void);
+
+/**
+ * @brief First fetches a byte from the address `HL`, subtracts the fetched byte from A
+ *        addtionally, subtract the carry flag, then store it in A.
+ */
```

Comment:
Maybe we could add a note here that `SBC A, (HL)` takes 8 cycles because it fetches a byte from memory? Same with `SUB A, (HL)`